mllib/src/test/scala/org/apache/spark/ml/classification/RandomForestClassifierSuite.scala

```diff
@@ -167,7 +167,8 @@ private object RandomForestClassifierSuite {
     val newModel = rf.fit(newData)
     // Use parent from newTree since this is not checked anyways.
     val oldModelAsNew = RandomForestClassificationModel.fromOld(
```

Comment:
In one of these tests, can you please test to make sure the "rawPrediction" is consistent with the "prediction" column?  There may be code in other tests suites you can reuse.  Also, please test the value of numClasses stored in the model.